core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -75,6 +75,9 @@ oppia.directive('topNavigationBar', [
           $scope.userMenuIsShown = ($scope.NAV_MODE !== NAV_MODE_SIGNUP);
           $scope.standardNavIsShown = (
             NAV_MODES_WITH_CUSTOM_LOCAL_NAV.indexOf($scope.NAV_MODE) === -1);
+          if ($scope.NAV_MODE === 'topics_and_skills_dashboard') {
```

Comment:
Why not just include this in NAV_MODES_WITH_CUSTOM_LOCAL_NAV?

Reply:
Done.